core/components/DatepickerInput.tsx

```diff
@@ -0,0 +1,21 @@
+import { type FC } from "react";
+
+type DatepickerInputProps = {
+  label: string;
+  value: string;
+  onChange: (date: string) => any;
+};
+
+export const DatepickerInput: FC<DatepickerInputProps> = ({ label, value, onChange }) => {
+  return (
+    <div className="block">
+      <label className="block text-lg font-medium text-gray-700">{label}</label>
+      <input
+        type="date"
```

Comment:
We can pass this as a prop and default it to date. Later it can be also used for datetime.